types/bindings/index.d.ts

```diff
@@ -1,13 +1,65 @@
-// Type definitions for bindings 1.3
+// Type definitions for bindings 1.5
 // Project: https://github.com/TooTallNate/node-bindings
-// Definitions by: Daniel Perez Alvarez <https://github.com/unindented>
+// Definitions by: Daniel Perez Alvarez <https://github.com/unindented>, Boris Gulay <https://github.com/BoresXP>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
+/// <reference types="node" />
+
+export interface BindingOptions {
+	/**
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/bindings/index.d.ts;
  Line: 9;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.